src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -101,20 +101,22 @@ private void replaceText(String text) {
     @FXML
     private void handleCommandInputChanged() {
         try {
+            final boolean isInvalid = false;
```

Comment:
This is a weird coding style. If you're using it as a constant, you might want to put it in as class constant, especially when you're doing it twice. Also, you can give more information, e.g. which parameter / what are you specifying as invalid?